src/Sylius/Behat/Resources/config/suites/ui/checkout/checkout.yml

```diff
@@ -11,13 +11,15 @@ default:
                 - sylius.behat.context.transform.lexical
                 - sylius.behat.context.transform.payment_method
                 - sylius.behat.context.transform.product
-                - sylius.behat.context.transform.shipping_method
                 - sylius.behat.context.transform.zone
+                - sylius.behat.context.transform.shared_storage
+                - sylius.behat.context.transform.shipping_method
```

Comment:
We should, unfortunately there is a conflict between ``the`` transformer from``SharedStorageContext`` and ``the rest of the world`` transformer from ``ZoneContext``. If ``sylius.behat.context.transform.shared_storage`` is included before ``sylius.behat.context.transform.zone``, part of step ``the rest of the world`` is identified as **resource from shared context with key "rest_of_the_world"**, not as **"Rest of the World" zone" ;/